hw3.py

```diff
@@ -1,22 +1,23 @@
-print('I love Python '*42)
+print('I love Python ' * 42)
 
 age_in_month = 327
 age_in_years = age_in_month // 12
 print(age_in_years)
 
-name='Nazar'
-my_age="My name is " + name + " I'm " + str(age_in_years) + " years old"
+name = git
```

Comment:
Changes in this file are not required by the task. I can approve it, bu only after fixing some error in this line: `name = git`